test/ui/driverBuilder.ts

```diff
@@ -0,0 +1,27 @@
+import { Builder, WebDriver } from 'selenium-webdriver';
+import 'chromedriver';
+import * as chrome from 'selenium-webdriver/chrome';
+
+export class driverBuilder {
+
+    static driver: WebDriver;
+
+    static async before(): Promise<WebDriver> {
```

Comment:
before/after are test concepts (ran before and after each test) - can these functions be renamed?